{ 'init_fall_front': { 'head': [],
                       'left_arm': [],
                       'left_leg': [ { 'duration': 1.0,
                                       'name': u'init_fall_front',
                                       'positions': [ -0.0009,
                                                      -0.0013,
                                                      0.0517,
                                                      0.0055,
                                                      0.3953,
                                                      0.0002],
                                       'starttime': 0.0}],
                       'right_arm': [],
                       'right_leg': [ { 'duration': 1.0,
                                        'name': u'init_fall_front',
                                        'positions': [ 0.0009,
                                                       0.0013,
                                                       -0.0517,
                                                       -0.0055,
                                                       -0.3953,
                                                       -0.0002],
                                        'starttime': 0.0}],
                       'torso': []}}
